.eslintrc.cjs

```diff
@@ -1,29 +1,18 @@
 module.exports = {
   env: {
-    browser: true,
```

Comment:
Not sure why this was here, but this package is not setup to work in a browser environment.